### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by 50Wliu**
Initial draft for the new issue template.

/cc @lee-dohm: Had to remove a lot of the Atom-specific stuff...doesn't look nearly as spectacular as the Atom one :P

### CODE REVIEW

**Comment by lee-dohm** [comment]
This is a great start ... thanks so much @50Wliu :metal:

**Comment by zcbenz** [comment]
IMO there are too much information in the template, from my experience only two things are necessary for most issues: Electron's version and the operating system.

So a simple template like this would work for me:

```
Electron version:
Operating system:
```

**Comment by baconface** [comment]
In my opinion making it a procedure with steps and/or requirements may deter some people from reporting. Especially if you are looking at taking 15-30 minutes to report a very simple or minor issue. And the people who do post low quality issues will likely not read/follow this in the first place. Of course, just my 2 cents.

**Comment by baconface** [comment]
What about making a debug/info dump file with error stack info, os info, Electron version, etc and ask them to post the contents if possible/when applicable?

**Comment by 50Wliu** [comment]
I just shortened the template...mind taking a look again @paulcbetts @zcbenz @baconface?

(Also, `[ci skip]` doesn't seem to stop the `electron-mas-x64` build :frowning:)

**Comment by zcbenz** [comment]
Adding requirements in template won't prevent people from posting garbages, they would just delete everything in the comment box.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### Prerequisites
+
+* [ ] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+repo%3Aatom%2Felectron) to see if it is already reported?
```
Comment (zcbenz) [inline ISSUE_TEMPLATE.md]: I think we can assume people have already searched existing issues before posting a new one, if they haven't, I doubt they would just ignore this.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+### Description
+
+[Description of the bug or feature, for example, what you were doing when the issue arose and what you expected to happen]
+
+Electron version:
```
Comment (zcbenz) [inline ISSUE_TEMPLATE.md]: Probably move electron version and OS above `[Description of bug]`, the description can be quite long.

**Comment by lee-dohm** [comment]
If you all think that an Issue template is unnecessary for the Electron repo, I'm totally cool with that :grinning:

**Comment by paulcbetts** [comment]
@lee-dohm I think the reminder to list what OS they're having problems with is legit, people always forget that

**Comment by lee-dohm** [comment]
So maybe something really simple like this?

**Comment by baconface** [comment]
I like it really simple like this. :+1: 

It could have a few more fields if needed. But this asks for the always needed information as well as makes it less noisy.

**Comment by zcbenz** [comment]
:heart: the simple one.

**Comment by lee-dohm** [comment]
Thanks @zcbenz!

**Pull request merged**